spec.html

```diff
@@ -23753,7 +23753,7 @@
 
       <!-- es6num="20.1.1.1" -->
       <emu-clause id="sec-number-constructor-number-value">
-        <h1>Number ( [ _value_ ] )</h1>
+        <h1>Number ( _value_ )</h1>
         <p>When `Number` is called with argument _number_, the following steps are taken:</p>
         <emu-alg>
           1. If no arguments were passed to this function invocation, let _n_ be +0.
```

Comment:
because of this line, `Number(undefined)` and `Number()` differ - this, imo the `value` argument should indeed remain default. It's kind of a weird case, of course, because absence and `undefined` are different, so I don't feel as strongly about this one.